### PR TITLE
Update `man maestro-config` to include a description

### PR DESCRIPTION
**Pull request by T-monius**
The description for the `maestro config` command on the manpages now includes info regarding when the command should be run in addition to what it does.

A user is guided to run the command directly after install and informed as to the directory (i.e. `.maestro`) created in the users home directory and the AWS account information file.

The user is informed as to the nature of the information the hidden account information file contains and the format required for that information. A link is provided regarding AWS regions that can be input.

### CODE REVIEW

**Comment by johnisom** [comment]
Looks like we both worked on the `manpages` branch. All is fine, though I will explain what changes I made so they can be reviewed as well:

I removed the `bin/zip-man-pages.sh` file, installed the non-gzipped manpages (updated in `package.json`), and removed the `preinstall` hook in `package.json`.

File: doc/man/maestro-config.1 [hunk 1]
```diff
@@ -12,6 +12,29 @@ maestro-config \- set up or alter your Maestro configuration files
...
+command should be the first command run after installing Maestro. Any Maestro deployment depends on the account information in the hidden directory created by the initial invocation of
+.I maestro config.
+
+.PP
+.I maestro config
+checks for creation of a hidden
+.I .maestro
+directory created in the users home directory and also the existence of a file
+.I aws_account_info.json
+. If the directory and file exists, and there is information present, the values present are presented to the user when prompted for account information. Otherwise, a user will simply be prompted for appropriate account information (i.e. in the case this is the initial invocation of the command).
```
Comment (johnisom) [inline doc/man/maestro-config.1:27]: I tested this, and weirdly enough, this won't render the "if the directory and file exists..." line (27). I think line 26 needs to be `.IR aws_account_info.json .`, then line 27 can just start with text (not period) like so:
```
.IR aws_account_info.json .
If the directory and file exists...
```
.

I think the reason for this is any period at the beginning of the line will try to be interpreted as a macro. If we put the period on the line before, the issue is resolved.

File: doc/man/maestro-config.1 [hunk 2]
```diff
@@ -12,6 +12,29 @@ maestro-config \- set up or alter your Maestro configuration files
...
+
+.PP
+The pertinent pieces of account information that
+.I maestro config
+is concerned with are the AWS account number and region. The account number is a 12 digit number, and the user is required to input said number in the format
+.I XXXXXXXXXXXX
+
```
Comment (johnisom) [inline doc/man/maestro-config.1:34]: There should probably be a period at the end of this sentence

File: doc/man/maestro-config.1 (hunk 1 above)
Comment (T-monius) [reply doc/man/maestro-config.1]: Nice catch. I'd actually started another line with a period and noticed the same error when I tested. I neglected that I'd used the same erroneous syntax on line 26.

File: doc/man/maestro-config.1 (hunk 2 above)
Comment (T-monius) [reply doc/man/maestro-config.1]: Added

**Pull request merged**